### DISCUSSION TITLE
Please Reconsider the Decision to Not Merge uBO into uBOL.

### DISCUSSION THREAD

**Discussion by AT-StephenDetomasi** [uBlockOrigin/uBOL-home#262] [General]
As someone who has deployed uBO for thousands of people since 2014, I am really saddened that it now appears to be blocked for most people who are running the latest version of Chrome, including myself. For a little while we could simply tell Chrome to keep the extension enabled, but this option is now gone on the latest version of Chrome 131.x 

According to the the FAQ, the plan it to simply let the extension stop working for Chrome users and instead let users work it out for themselves to install another extension: https://github.com/uBlockOrigin/uBOL-home/wiki/Frequently-asked-questions-(FAQ)#will-ubo-automatically-transition-to-ubo-lite-in-the-chrome-web-store

The vast majority of the people I have deployed uBO for over the years have absolutely loved not having ads on their computer, but many of them are vulnerable users, typically of the older generation, that don't possess the technical know-how to install a replacement browser extension on their own. I understand that uBOL isn't as good as uBO, but something is better than nothing, and a fear a lot of repercussions from scams and malicious ads for these users. As such, I'm calling upon the creators to strongly reconsider this decision.

I am not saying that any changes need to be made for Firefox users, and a separate extension could be created for Chromium browser users who are able to use Manifest V2 (e.g. Brave browser users), but I am strongly urging the creators of uBO to reconsider the decision to let the extension die. Whilst you and I reading this can easily switch extensions, it's also you and I who are also being called upon to fix their technical problems for the less tech-savvy, and it cannot be understated how valuable uBO has been for preventing issues in the first place.

Thank you for reading my proposal.

**Comment by MasterKia** [comment]
https://github.com/gorhill/uBlock/wiki/About-Google-Chrome's-%22This-extension-may-soon-no-longer-be-supported%22:

> Manifest v2 uBO will not be automatically replaced by Manifest v3 uBOL. uBOL is too different from uBO for it to silently replace uBO -- **_you will have to explicitly make a choice as to which extension should replace uBO according to your own prerogatives._**
Ultimately whether uBOL is an acceptable alternative to uBO is up to you, **_it's not a choice that will be made for you._**

**Comment by AT-StephenDetomasi** [comment]
Would it perhaps instead be possible for the Ublock Origin extension on Chrome to provide a notice or information about the changes to help them at least make a choice? Many end users ignore things that they don't understand. End users are not given a choice by the extension itself - the Ublock Origin extension simply stops working on Chrome, at which point the user gets bombarded with ads. They may end up doing a Bing search for "adblock" and will likely end up installing some sponsored garbage. It's unlikely they will even realise that there us a Ublock Origin Lite, or that Firefox supports the full uBlock Origin experience. We are talking about average users here, not power users like ourselves who have a github account.
